src/lib.rs

```diff
@@ -136,117 +144,74 @@ fn parse_airport(network_list: &str) -> Result<Vec<Wifi>, Error> {
     Ok(wifis)
 }
 
-#[allow(dead_code)]
-fn parse_iwlist(network_list: &str) -> Result<Vec<Wifi>, Error> {
-    let mut wifis: Vec<Wifi> = Vec::new();
-
-    let cell_regex = try!(Regex::new(r"Cell [0-9]{2,} - Address:")
-                              .map_err(|_| Error::SyntaxRegexError));
-
-    let mac_regex =
-        try!(Regex::new(r"([0-9a-zA-Z]{1}[0-9a-zA-Z]{1}[:]{1}){5}[0-9a-zA-Z]{1}[0-9a-zA-Z]{1}")
-                 .map_err(|_| Error::SyntaxRegexError));
-
-    for block in cell_regex.split(&network_list) {
-        let mut lines = block.lines();
-
-        let mut wifi_mac = String::new();
-        let mut wifi_ssid = String::new();
-        let mut wifi_channel = String::new();
-        let mut wifi_rssi = String::new();
-        let wifi_security = String::new(); // FIXME needs implementing
-
-        let mac_matches = mac_regex.captures(try!(lines.next().ok_or(Error::NoValue)));
+fn parse_iw_dev(interfaces: &str) -> Result<String, Error> {
```

Comment:
why is there suffix of `_dev` against this function? is this the full name of the replacement for `iwlist`?